src/test/java/bio/terra/service/snapshot/SnapshotDaoTest.java

```diff
@@ -319,7 +319,8 @@ public void snapshotEnumerateTest() throws Exception {
             null, "==foo==", null, datasetIds, snapshotIdList);
         List<SnapshotSummary> summaryList = summaryEnum.getItems();
         assertThat("filtered and retrieved 2 snapshots", summaryList.size(), equalTo(2));
-        assertThat("filtered total 3", summaryEnum.getTotal(), equalTo(3));
+        assertThat("filtered total 3", summaryEnum.getFilteredTotal(), equalTo(3));
+        assertThat("total 6", summaryEnum.getTotal(), equalTo(6));
```

Comment:
Cool.  Probably need an equivalent test in DatasetDaoTest

Reply:
Added cases to `DatasetDaoTest`